test/views/html/pages/ChatViewBehaviours.scala

```diff
@@ -68,4 +68,23 @@ trait ChatViewBehaviours extends ViewSpecBase {
       }
     }
   }
+
+  def generalContent(view: () => HtmlFormat.Appendable,
+                     messageHeading: String,
+                     betaBannerText: String
+                    ): Unit = {
+    "adds to a all pages" when {
+      "display the correct page title" in {
+        val doc = asDocument(view())
+        doc.getElementsByTag("h1")
+        assertPageTitleEqualsMessage(doc, s"$messageHeading")
+      }
+
+      "display the beta banner when flag is true" in {
```

Comment:
Should it be just "display the beta banner"? as there is no flag

Reply:
removed "when flag is true"